webapp/src/component/navigation/NavigationWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import grey from '@material-ui/core/colors/grey';
+import { Box, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  container: {
+    height: 48,
+    display: 'flex',
+    alignItems: 'center',
+    '& > * + *': {
+      marginLeft: 10,
```

Comment:
Do you really need exact 10px? Component Box has "ml" or "marginLeft" prop or you can use theme.spacing function.